embedded-iconoir/src/icon.rs

```diff
@@ -50,18 +47,17 @@ make_icon!(SomeIcon, 24, "Animals", "fish");
 pub struct Icon<C, Ico>
 where
     C: PixelColor,
-    Ico: IconoirInternal,
+    Ico: sealed::IconoirInternal,
 {
     color: C,
-    #[allow(unused)]
-    icon: Ico,
+    _icon: Ico,
 }
 
-impl<C: PixelColor, Ico: IconoirInternal> Icon<C, Ico> {
+impl<C: PixelColor, Ico: sealed::IconoirInternal> Icon<C, Ico> {
     pub fn new(color: C) -> Self {
         Self {
             color,
-            icon: Ico::_new_internal(),
+            _icon: Ico::INSTANCE,
```

Comment:
This is a little "off topic", though since you can't have const fns in traits, but can have const vals, I thought this might be nice. That said, it's a ZST anyway, so... lemme know if you want me to back this change out :)

Reply:
I like your change. It's more elegant that way, especially because it completely removes functions from the internal trait, which is very elegant